addons/website_event_exhibitor/views/event_exhibitor_templates_list.xml

```diff
@@ -160,17 +160,17 @@
                 <button class="accordion-button collapsed"
                     type="button"
                     data-bs-toggle="collapse"
-                    data-bs-target=".o_wevent_offcanvas_country"
+                    data-bs-target=".o_wevent_offcanvas_level"
                     aria-expanded="false"
-                    aria-controls="o_wevent_offcanvas_country">
+                    aria-controls="o_wevent_offcanvas_level">
                     By Level
                 </button>
             </h2>
-            <div class="o_wevent_offcanvas_country accordion-collapse collapse">
-                <div class="accordion-body pt-0">
+            <div class="o_wevent_offcanvas_level accordion-collapse collapse">
+                <div class="accordion-body bg-white pt-0">
                     <ul class="list-group list-group-flush">
                         <li class="list-group-item">
-                            <a >
+                            <a class="o_dropdown_reset_tags cursor-pointer">
```

Comment:
Users and assistive technologies can't reach to `<a>` tag without `href` attribute. 
Could you add `href="#"` and `title`? Thanks!